# double array
pose_publisher:
  ros__parameters:
    number_of_links: 3
    links_lenght: [0.1, 0.2, 0.3]
